src/ngraph/runtime/gpu/gpu_backend.hpp

```diff
@@ -36,6 +36,13 @@ namespace ngraph
                 std::shared_ptr<ngraph::runtime::TensorView>
                     make_primary_tensor_view(const ngraph::element::Type& element_type,
                                              const Shape& shape) override;
+
+                bool compile(const std::shared_ptr<ngraph::Function>& fun) override;
+
+                bool is_callable() const override;
```

Comment:
I'm surprised this didn't fail at compile time since you removed the inherited function call.